Adds the last batch of bugs to the RNs (#48597)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

* Previously, when setting up `br-ex` bridge on boot, the DHCP client id and IPv6 address generation mode configuration did not work properly causing an unexpected IP address on `br-ex`. With this update, DHCP client id and IPv6 generation mode configuration are now properly set on `br-ex`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2058030[*BZ#2058030*])

> * When an OVN cluster has more than 75 worker nodes, simultaneously creating 2000 or more services and route objects can cause pods created at the same time to hang in the `ContainerCreating` status. If this problem occurs, entering the `oc describe pod <podname>` command will show events with the following warning: `FailedCreatePodSandBox...failed to configure pod interface: timed out waiting for OVS port binding (ovn-installed)'. There is currently no workaround for this issue.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084062[*BZ#2084062*])

Author: ack. Must have missed this. Moving it!

* Previously, `egress-router-cni` lacked some cluster internal routes due to the reliance on the `gateway` field of the CNI definition to delete the default route and inject its own. With this update, `egress-router-cni` pods inject correct routing information so that external and internal cluster destinations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2075475[*BZ#2075475*])

> * There is currently a known issue with OVN-Kubernetes that whenever the `NetworkManager` service restarts the node will lose network connectivity and must be recovered. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2074009[*BZ#2074009*])

Reviewer: ```suggestion
* There is currently a known issue with OVN-Kubernetes that whenever the `NetworkManager` service restarts, the node will lose network connectivity and must be recovered. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2074009[*BZ#2074009*])
```

Should this go with the known issues?

Author: yes! TY!

> * Previously, a `PodDisruptionbudget` was used to create a OVN raft quorum on {SNO}. This raised an unhelpful `PodDisruptionBudgetAtLimit` on {SNO} clusters. With this update, the `PodDisruptionBudgetAtLimit` is no longer raised on these clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037721[*BZ#2037721*])

Reviewer: ```suggestion
* Previously, a pod disruption budget was used to create a OVN raft quorum on {SNO}. This raised an unhelpful `PodDisruptionBudgetAtLimit` alert on {SNO} clusters. With this update, the `PodDisruptionBudgetAtLimit` alert is no longer raised on these clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037721[*BZ#2037721*])
```

> * Previously, a race condition in `NetworkManager` restart sometimes prevented DHCP resolution to successfully complete setting up `br-ex` bridge on node boot when using OVN-Kubernetes. With this update, `NetworkManager` is no longer restarted when setting up `br-ex` eliminating the race condition. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055433[*BZ#2055433*])

Reviewer: ```suggestion
* Previously, a race condition in the `NetworkManager` restart sometimes prevented DHCP resolution to successfully complete setting up the `br-ex` bridge on node boot when using OVN-Kubernetes. With this update, `NetworkManager` is no longer restarted when setting up `br-ex`, eliminating the race condition. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055433[*BZ#2055433*])
```

* There is currently a known issue with OVN-Kubernetes that whenever the `NetworkManager` service restarts the node will lose network connectivity and must be recovered. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2074009[*BZ#2074009*])

* Previously, a `PodDisruptionbudget` was used to create a OVN raft quorum on {SNO}. This raised an unhelpful `PodDisruptionBudgetAtLimit` on {SNO} clusters. With this update, the `PodDisruptionBudgetAtLimit` is no longer raised on these clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037721[*BZ#2037721*])

* Previously, a race condition in `NetworkManager` restart sometimes prevented DHCP resolution to successfully complete setting up `br-ex` bridge on node boot when using OVN-Kubernetes. With this update, `NetworkManager` is no longer restarted when setting up `br-ex` eliminating the race condition. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055433[*BZ#2055433*])

> *

Reviewer: ```suggestion

```

* Previously, the output for the `oc explain router.status.ingress.conditions` command explain route status showed `Currently only Ready` rather than `Admitted` due to incorrect wording in the Application Programming Interface (API). This fix corrects the wording in the API. As a result, the command output is correct. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2041133#[*BZ#2041133*])

> * Previously,  the Ingress Operator detected that the user modified an annotation that the Operator manages on the `LoadBalancer-type` service. Consequently, the Operator sets the Ingress Cluster Operator's `Upgradeable` status condition `False` to block upgrades. and the Ingress Operator erroneously set the `Upgradeable` status condition to `False`, blocking upgrades, if the service had no annotations. Fix: The logic that checks the service's annotations was fixed to handle empty annotations correctly. Result: The ingress operator should no longer erroneously block upgrades. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097555[*BZ#2097555*])

Reviewer: ```suggestion
* Previously, the Ingress Operator detected that the user modified an annotation that the Operator manages on the `LoadBalancer-type` service. Consequently, the Operator set the Ingress Cluster Operator's `Upgradeable` status condition to `False` to block upgrades, and the Ingress Operator erroneously set the `Upgradeable` status condition to `False`, blocking upgrades, if the service had no annotations. Now, the logic that checks the service's annotations correctly handles empty annotations, and the Ingress Operator no longer erroneously blocks upgrades. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097555[*BZ#2097555*])
```

* Previously, the Ingress Operator removed a finalizer that the Operator added to `LoadBalancer-type` services from previous versions of {product-title}. With this update, the Ingress Operator no longer removes finalizers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069457[*BZ#2069457*])

> * The Ingress Operator performs health checks against the ingress canary route. Berfore this update,  keepalives are enabled on the connection causing the Ingress Operator not to close the TCP Connection to the load balancer (LB) once the health check was completed. A new connection was created for the next health check instead of using the existing connection. Consequently, the connection built on the LB and creating too many connections on the LB. With this update,  keepalives are disabled when connecting to the canary route, and a new connection is made and closed each time the canary probe is run. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037447[*BZ#2037447*])

Reviewer: ```suggestion
* The Ingress Operator performs health checks against the ingress canary route. Before this update, `keepalive` daemons are enabled on the connection, which caused the Ingress Operator not to close the TCP Connection to the load balancer (LB) after the health check was completed. A new connection was created for the next health check instead of using the existing connection. Consequently, the connection built on the load balancer and creating too many connections on the load balancer. With this update, `keepalive` daemons are disabled when connecting to the canary route, and a new connection is made and closed each time the canary probe is run. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037447[*BZ#2037447*])
```

> * Previously, the Ingress Controller did not set the `allowPrivilegeEscalation` to false in the router deployment, causing the router pods to get selected into the incorrect Security Context Constraint (SCC) and creating conflicts with custom SCCs. This fix sets the `allowPrivilegeEscalation` to true, ensuring that router pods get selected into the correct SCC and avoid conflicts with custom SCCs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2007246#[*BZ#2007246*])

Reviewer: ```suggestion
* Previously, the Ingress Controller did not set the `allowPrivilegeEscalation` value to false in the router deployment, which caused the router pods to be selected into the incorrect Security Context Constraint (SCC) and created conflicts with custom SCCs. This fix sets the `allowPrivilegeEscalation` value to `true`, ensuring that router pods are selected into the correct SCC and avoid conflicts with custom SCCs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2007246#[*BZ#2007246*])
```

> * Previously when the canary route was not admitted to an ingress controller, the ingress operator status condition did not show as `degraded`. Consequently, the canary route could have shown as `valid` when its status condition should show as `not admitted`. With this update, the ingress operator status more accurately reflects the status of the canary controller. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2021446#[*BZ#2021446*])

Reviewer: ```suggestion
* Previously, when the canary route was not admitted to an Ingress Controller, the Ingress Operator status condition did not show as `degraded`. Consequently, the canary route could have shown as `valid` when its status condition should show as `not admitted`. With this update, the Ingress Operator status more accurately reflects the status of the canary controller. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2021446#[*BZ#2021446*])
```

> * Previously, the openshift-router process briefly ignored the SIGTERM shutdown signal. This caused containers to ignore a kubernetes shutdown request resulting in one hour shutdown times. With this update, the router now responds to SIGTERM signals. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076297#[*BZ#2076297*])

Reviewer: ```suggestion
* Previously, the `openshift-router` process briefly ignored the `SIGTERM` shutdown signal. This caused containers to ignore a Kubernetes shutdown request, resulting in one hour shutdown times. With this update, the router now responds to `SIGTERM` signals. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076297#[*BZ#2076297*])
```

* This fix ensures that read-only-many volumes are provisioned appropriately by GCP CSI Driver as read-only. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1968253#[*BZ#1968253*])

> * {product-title} allows installation of of vSphere CSI drivers shipped by either the upstream community or VMware. Although Red Hat does not support this driver, cluster administrators ca still install and use it because it has more features than the vSphere CSI driver shipped by Rd Hat. {product-title} can be upgraded to {product-version} with the upstream and VMware vSphere CSI driver, however, customers will be warned about the presence of a third party CSI driver. In a future release {product-title}, customers will be required to upgrade to the version of vSphere CSI shipped by Red Hat before upgrading to the next version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2089419[BZ#2089419] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2052071[*BZ#2052071*]).

Reviewer: ```suggestion
* {product-title} allows installation of of vSphere CSI drivers shipped by either the upstream community or VMware. Although Red Hat does not support this driver, cluster administrators can still install and use it because it has more features than the vSphere CSI driver shipped by Red Hat. {product-title} can be upgraded to {product-version} with the upstream and VMware vSphere CSI driver, however, you will be warned about the presence of a third party CSI driver. In a future release {product-title}, you will be required to upgrade to the version of vSphere CSI shipped by Red Hat before upgrading to the next version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2089419[BZ#2089419] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2052071[*BZ#2052071*]).
```
